signalfx-yammer/src/main/java/com/signalfx/codahale/reporter/MetricMetadataImpl.java

```diff
@@ -0,0 +1,141 @@
+package com.signalfx.codahale.reporter;
```

Comment:
Also appears to be identical to https://github.com/signalfx/signalfx-java/blob/master/signalfx-codahale/src/main/java/com/signalfx/codahale/reporter/MetricMetadataImpl.java